pkg/project/v2/topologysort/topologysort.go

```diff
@@ -189,7 +189,7 @@ func parametersToSortData(conf coordinate.Coordinate, parameters []ParameterWith
 			}
 
 			if p.IsReferencing(conf, param) {
-				log.Debug("\t\t%s has dep on %s", p.Name, param.Name)
+				logDependency(p.Name, param.Name)
```

Comment:
Not sure if this log line is meaningful? It will print some config internal dependencies without giving any idea about the context.

Reply:
I think it's most certainly interesting debug information to us to see if dependency resolution works as it should? 

It now logs a bit more understandable "%s has dependency on %s" as well.

I'd keep it. What do you think?

Reply:
Oh I'm not saying we need to remove it, but just to give it a bit mode context. As you can see in my example I'm missing the config type information. I only see there is some dependency but still need to search to actually realise which config is depending on what.

Reply:
Log now looks like this in verbose mode

```
2021/10/06 13:26:23 INFO  Dynatrace Monitoring as Code v2.0.0-next
2021/10/06 13:26:23 DEBUG Loading configuration projects ...
2021/10/06 13:26:23 DEBUG Project: ims has dependency on infrastructure
2021/10/06 13:26:23 DEBUG Project: ada has dependency on infrastructure
2021/10/06 13:26:23 DEBUG Configuration: infrastructure:management-zone:zone has dependency on infrastructure:auto-tag:application-tagging
2021/10/06 13:26:23 DEBUG Configuration: infrastructure:management-zone:zone has dependency on infrastructure:aws-credentials:aws
2021/10/06 13:26:23 DEBUG Configuration: infrastructure:synthetic-monitor:ingressInternalAvailability has dependency on infrastructure:auto-tag:infra-tagging
2021/10/06 13:26:23 DEBUG Configuration: infrastructure:synthetic-monitor:ingressPublicAvailability has dependency on infrastructure:auto-tag:infra-tagging
2021/10/06 13:26:23 DEBUG Configuration: infrastructure:alerting-profile:profile has dependency on infrastructure:auto-tag:infra-tagging
2021/10/06 13:26:23 DEBUG Configuration: infrastructure:dashboard:ingress-internal has dependency on infrastructure:synthetic-monitor:ingressInternalAvailability
2021/10/06 13:26:23 DEBUG Configuration: infrastructure:dashboard:ingress-public has dependency on infrastructure:synthetic-monitor:ingressPublicAvailability
2021/10/06 13:26:23 DEBUG Configuration: infrastructure:notification:email has dependency on infrastructure:alerting-profile:profile
2021/10/06 13:26:23 DEBUG Configuration: infrastructure:notification:slack has dependency on infrastructure:alerting-profile:profile
```